packages/compose/src/hooks/use-global-event/index.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * WordPress dependencies
+ */
+import { useEffect } from '@wordpress/element';
+
+export default function useGlobalEvent( ref, ...args ) {
```

Comment:
Somehow I'd like the leave options open for the use of all `addEventListener` parameters.

Reply:
Instead of rest parameters, why not do something like this:

```js
useGlobalEvent( ref, addEventListenerArgs, dependencies )
```

`addEventListenerArgs` would be an array.

Reply:
Yeah, that sounds good too, though not as pretty. :D For the sake of the docs, we should do it. :)

Reply:
Ah, actually I like it.